frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -267,41 +241,89 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
             actions.stopAnimation()
         },
         setSpeed: ({ speed }) => {
-            values.replayer?.setConfig({ speed })
+            values.player?.replayer?.setConfig({ speed })
         },
-        seek: async ({ time, forcePlay }, breakpoint) => {
-            // Real seeking is debounced so as not to overload rrweb.
-            await breakpoint(100)
```

Comment:
Heads up, I ended up removing this breakpoint because `seek()` isn't actually called that often, but in removing it, it reveled a set of race conditions around buffering + rr-web, so a bunch of the recent changes were to address that.